cpp/tests/test_ide_parameters_io.cpp

```diff
@@ -0,0 +1,124 @@
+/* 
+* Copyright (C) 2020-2024 MEmilio
+*
+* Authors: Lena Ploetzke
+*
+* Contact: Martin J. Kuehn <Martin.Kuehn@DLR.de>
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "ide_secir/infection_state.h"
+#include "ide_secir/model.h"
+#include "ide_secir/parameters_io.h"
+
+#include "memilio/config.h"
+#include "memilio/utils/date.h"
+#include "memilio/io/io.h"
+
+#include "test_data_dir.h"
+#include "load_test_data.h"
+#include "matchers.h"
+#include <gtest/gtest.h>
+
+// Check that initialization based on synthetic RKI data match previous result.
+TEST(TestIDEParametersIo, RKIcompareWithPreviousRun)
+{
+    // Define start date and the total population used for the initialization.
+    ScalarType total_population = 80 * 1e6;
+    auto start_date             = mio::Date(2020, 6, 3);
+    ScalarType dt               = 0.5;
+    // Initialize model.
+    mio::isecir::Model model(mio::TimeSeries<ScalarType>(0., mio::TimeSeries<ScalarType>::Vector::Constant(
+                                                                 (int)mio::isecir::InfectionTransition::Count, 1.)),
+                             total_population, -1);
```

Comment:
Why do you set total_confirmed_cases to -1?

Reply:
Actually, i should just use the default value for this.